pages/common/views/components/inset.jsx

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
+import { castArray } from 'lodash';
+import classnames from 'classnames';
+
+const concatClassNames = fromProps => {
+  return castArray(fromProps).concat(['panel', 'panel-wide']).reduce((all, item) => {
+    return { ...all, [item]: true };
+  }, {});
+};
 
 const Inset = ({
-  children
+  children,
+  className
 }) => (
-  <div className="panel panel-border-wide">
+  <div className={classnames(concatClassNames(className))}>
```

Comment:
We don't need the `concatClassNames` method at all here - `classnames` can receive an array or rest args.

This will work fine:

```
<div className={classnames('panel', 'panel-wide', className)}>
```

Alternatively sack off `classnames` altogether and just:

<div className={['panel', 'panel-wide', className].join(' ')}>

Reply:
Ignore that last one - it could end up with `panel panel-wide undefined`

Reply:
good shout